kanban-all/kanban-data/src/main/java/ru/ifmo/p3411/repositories/TaskRepository.java

```diff
@@ -0,0 +1,14 @@
+package ru.ifmo.p3411.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.stereotype.Repository;
+import ru.ifmo.p3411.data.Task;
+
+/**
+ * @author Ellepsis
+ * @since 1.0
+ */
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long>, JpaSpecificationExecutor<Task> {
```

Comment:
Please add a **smoke test** in the test directory:

- Set up test Spring context 
- Create all necessary beans inside the context configuration class, e.g. fake do-nothing datasource (or even in-memory H2 if you prefer it!)
- Write @Test with empty body
- Maybe additional steps...?

This is really useful to see if the context starts up properly and without errors, and will be done on each TeamCity build!

I'd expect you to use 
@RunWith(SpringRunner.class)
@WebAppConfiguration
@ContextConfiguration 

etc. see https://docs.spring.io/spring/docs/current/spring-framework-reference/html/integration-testing.html